gulpfile.js

```diff
@@ -141,7 +143,21 @@ function lintTask() {
     rules: {
       'complexity': [1, 6],
       'max-statements': [1, 30]
-    }
+    },
+    globals: [
```

Comment:
do you think any problems will occur from having the globals defined for the src as well?